### ISSUE TITLE
Double channel list

### ISSUE THREAD

**Issue by banditkroot** [cryzed/Twitch-IRC-Userlist-Fix#1]
I'm using xchat on Linux, and sometime on small channel, the userlist load two times and I've each member twice in the userlist.
I've also double join and double part message for the user.

**Comment by cryzed** [comment]
Are you sure you are using the latest version of the script? If you are, it would be great if you could try HexChat instead of X-Chat, which is a more up-to-date and free fork of X-Chat. I was a bit sparse with checks and assertions, since I never saw duplicated users in my list myself.

**Comment by banditkroot** [comment]
Yes I'm using the latest version of the script. I tried it on Hexchat, after two crash of Hexchat (not because of your script), it seems that the problems don't appear in Hexchat. I'll use Hexchat a little more for your script, but I think that Hexchat isn't very stable...

**Comment by cryzed** [comment]
I'll try and see if I can implement an userlist check before generating part and join events in the coming days, then it hopefully will also work properly with X-Chat.
